wild_lib/src/args.rs

```diff
@@ -139,7 +139,13 @@ const SILENTLY_IGNORED_FLAGS: &[&str] = &[
     "relax",
 ];
 
-const IGNORED_FLAGS: &[&str] = &["gdb-index", "no-relax", "disable-new-dtags"];
+const IGNORED_FLAGS: &[&str] = &[
```

Comment:
Adding the flags here will mean that we issue a warning when those flags are supplied. Was that your intention? Given that the flags should be no-ops, perhaps we shouldn't issue a warning? Some of the flags above are probably no-op flags too, like "relax" and possibly "no-call-graph-profile-sort". Perhaps the new ones here should be grouped with those.

Reply:
Yeah, sorry, I forgot to test my changes :) It was not intentional.